go.mod

```diff
@@ -10,5 +10,6 @@ require (
 	github.com/golang/protobuf v1.4.2
 	github.com/pkg/errors v0.9.1
 	github.com/sourcegraph/lsif-protocol v1.0.0
+	github.com/sourcegraph/src-cli v0.0.0-20210115142024-761c534fc617 // indirect
```

Comment:
If you ran `go get` in this directory it will add that target to the project. If this was unrelated to an import (looks like it is, and nothing in src-cli repo is usefully importable anyway), you can undo this by running `go mod tidy` which will prune unused deps.

Reply:
That explains the diffs in this file! They looked unrelated but I wasn't sure what exact command updated these files.